WordPress/src/main/java/org/wordpress/android/ui/posts/PostListMainViewModel.kt

```diff
@@ -150,12 +150,6 @@ class PostListMainViewModel @Inject constructor(
     private val _onFabClicked = MutableLiveData<Event<Unit>>()
     val onFabClicked: LiveData<Event<Unit>> = _onFabClicked
 
-    private val _onFabLongPressedForCreateMenu = MutableLiveData<Event<Unit>>()
-    val onFabLongPressedForCreateMenu: LiveData<Event<Unit>> = _onFabLongPressedForCreateMenu
-
-    private val _onFabLongPressedForPostList = MutableLiveData<Event<Unit>>()
-    val onFabLongPressedForPostList: LiveData<Event<Unit>> = _onFabLongPressedForPostList
-
```

Comment:
Is the intent here to also remove the long-press toast?

Reply:
Reverted this to keep it on. 👍